examples/javascript-V4Auth-example.js

```diff
@@ -1,5 +1,5 @@
 const http = require('http');
-const aws4 = require('aws4');
+const aws4 = require('aws4');  // eslint-disable-line
```

Comment:
Seems like maybe we should just disable linting of the examples directory if that's not difficult to do.

Reply:
I like the idea :) Added changes to ignore the `examples` dir

Reply:
Seems to be a linter error
```
/home/ubuntu/utapi/examples/javascript-V4Auth-example.js
  2:22  error  Unable to resolve path to module 'aws4'  import/no-unresolved

```